src/common.cc

```diff
@@ -534,7 +545,7 @@ rd_kafka_NewTopic_t* FromV8TopicObject(
           err = rd_kafka_NewTopic_set_config(
             new_topic, pKeyString.c_str(), pValString.c_str());
 
-          if (err != RdKafka::ERR_NO_ERROR) {
+          if (err != RD_KAFKA_RESP_ERR_NO_ERROR) {
```

Comment:
What's the reason for doing this?

Reply:
Type mismatch. `err` is type `rd_kafka_resp_err_t`, `RdKafka::ERR_NO_ERROR` is of type `enum ErrorCode`, they are numerically equivalent, but not comparable (without warning).